universal/include/userver/utils/small_string.hpp

```diff
@@ -247,6 +255,13 @@ void SmallString<N>::push_back(char c) {
   data_.push_back(c);
 }
 
+template <std::size_t N>
+void SmallString<N>::append(std::string_view str) {
+  std::size_t old_size = data_.size();
+  data_.resize(old_size + str.size(), boost::container::default_init);
+  std::memcpy(data_.data() + old_size, str.data(), str.size());
```

Comment:
There should be `insert(begin, end)` or something like that in `smallvector`, does it perform worse here?

Reply:
I found no difference in performance. Still made the switch to insert() though.